opencog/cython/opencog/type_constructors.pyx

```diff
@@ -0,0 +1,132 @@
+
+# DO NOT EDIT THIS FILE! This file was automatically generated from atom
+# definitions in types.script by the macro OPENCOG_ADD_ATOM_TYPES
```

Comment:
On adding the TruthValue, you can call any Atom functions on each of the parts as returned by each function. So:

```
ConceptNode("dog").tv(TruthValue(0.5,0.5))
```
should work fine, but I haven't yet tested it. If it doesn't, I'll fix it up so it works well.

I can add more convenience functions if there are common ways of initializing TruthValue objects. So perhaps something like:

```
ConceptNode("dog").truth_value(0.5,0.5)
```